src/foam/nanos/doc/DocumentationView.js

```diff
@@ -52,6 +60,11 @@ foam.CLASS({
   ],
 
   methods: [
+    function init() {
+      var docId = (v => v.length > 1 ? v[1] : null)(
```

Comment:
I think being a little less clever would make this more readable. Just use a tmp var (which is what you're doing with 'v' anyway).

Reply:
Yeah my code here wound up much simpler after that - thanks!